src/wanna/cli/plugins/common_options.py

```diff
@@ -26,6 +26,13 @@
     "Use all for dev",
 )
 
+skip_containers_option = typer.Option(
+    False,
+    "--skip-containers",
```

Comment:
we already have a `PushMode` interface in pipelines/Jobs, can this be reused ?

perhaps a `PushMode.skip` or reuse the `PushMode.quick`

Reply:
Sounds good, will change :)